packages/compiler/server/serverlib.ts

```diff
@@ -1132,7 +1132,7 @@ export function createServer(host: ServerHost): Server {
     let dir = getDirectoryPath(path);
     const options = { allowFileNotFound: true };
 
-    while (inWorkspace(dir)) {
```

Comment:
Is there a chance that we get into an infinite loop here someway, it happened before but don't remember if the fix was in `inWorkspace` function or somewhere else.

Think it was happening on linux as `getDirname of "/" -> "" -> "" -> ""`

Reply:
Ah yeah that seems familiar